examples/streaming/README.md

```diff
@@ -166,10 +176,18 @@ endpoint, payload, err := cli.ParseEndpoint(
   goahttp.ResponseDecoder,
   debug,
   myDialer,
-  myConnConfigurer,
+  nil,
```

Comment:
We should group all these into one struct containing one field per configurer similar to what we do with endpoints. The generated code should also product a `NewConfigurers` function similar to the `NewEndpoints` function that accepts a single configurer and produces the struct with each field initialzed with that configurer. This way the user can simply create the server struct with something like `New(chatterEndpoints, mux, dec, enc, eh, myUpgrader, NewConfigurers(myconfig))`. The user can also create the configurers struct and override each field prior to creating the server if need be.